effect/std/atoms/rpcClient.ts

```diff
@@ -2,14 +2,17 @@ import * as rpc from "../../../rpc/mod.ts";
 import { effector, EffectorArgs } from "../../impl/mod.ts";
 
 export interface RpcClientR<Beacon> {
-  rpc: rpc.RpcClientFactory<Beacon>;
+  rpc: rpc.RpcClientFactory<Beacon, rpc.RpcError>;
```

Comment:
Ignore the fact that the RPC errors aren't piped in through the top-level `rpcClient` effect. We'll get to this later (#46).